doc/pages/highlighters.asciidoc

```diff
@@ -51,6 +51,9 @@ highlighter is replaced with the new one.
     *-tabpad* <separator>:::
         a one character long separator that will be appended to tabulations to honor the *tabstop* option
 
+    *-guide* <separator>:::
+        a one character long separator that, if provided, will replace every *indentwidth* whitespace char
```

Comment:
Does it replace *every* whitespace char, or only consecutive leading spaces?

Reply:
consecutive leading spaces/tabs. Although I mentioned in a comment that there's a caveat. Current implementation only iterates over what's visible on screen.